### DISCUSSION TITLE
Total time of videos or option to snapshots

### DISCUSSION THREAD

**Discussion by deceivorz** [roleoroleo/yi-hack-Allwinner#292] [Ideas]
Hello,

First of all very thanks for your work and to share it. I want to set the motion recorded video to 10 secs unstead of current 60secs. Is some way to do it? I found the value 'TIME=60' inside the file record.sh. Tried to modify this and dont worked. Maybe take snapshots every second when motion is detected  can be another solution.

This is because i am thinking use the camera as  surveillance system and if it take 1 min to finish/upload the video they can disconnect the camera before the video finish.

Regards.

**Comment by roleoroleo** [comment]
Which records?
If you mean the records saved automatically from the cam it's not possible. It's managed from the original yi app.
The script is executed only if you start a record using web services.

**Reply by deceivorz** [reply]
Thanks for your reply. so is no way to set the max time of videos in 10 seconds? Or take snapshots when motion detected? :(
